### DISCUSSION TITLE
External sensor unable to send logs back to manager

### DISCUSSION THREAD

**Discussion by Hammer-Bear** [Security-Onion-Solutions/securityonion#7600] [Unsupported Versions] [answered]
Got an issue with an external sensor unable to send the logs back to the manager, it's sending grafana data back as I can see it in the dashboard and grid says its fine, but it's not sending alerts back to the manager, looking at the logs and the only thing that seems to be having problems is `/opt/so/log/filebeat/filebeat.log ` `2022-03-17T14:43:06.683Z        ERROR   [publisher_pipeline_output]     pipeline/output.go:154  Failed to connect to backoff(async(tcp://XXX-MNGR01:5644)): dial TCP (internal address):5644: connect: no route to host` so it looks like its trying to go back to its internal address which it wont be able to reach as its an external sensor, I can, however, ping the external address/dns name from the sensor, where can I change this to point to the external? I've tried testing the filebeat.yml but I assume that gets overwritten from salt or something as it keeps reverting back to the internal address.

I've also tried editing the minion file under `/opt/so/saltstack/local/pillar/minions/(hostname)_sensor.sls` but no avail and just keeps reverting back to the internal address.

**Comment by weslambert** [comment]
If this is your only sensor, have you tried copying the default filebeat.yml to the local equivalent (on the manager) and modifying there? Otherwise, have you tried using a VPN?

**Reply by Hammer-Bear** [reply]
We have an internal sensor and this one which is on a remote site, we've tried modifying the filebeat.yml from the manager but it then breaks the internal sensor as it tries to go out and back in. VPN probably isn't an option either as we were looking to put more sensors out on remote sites that wont have the ability to set a VPN up back to manager

**Comment by Hammer-Bear** [comment, marked as answer]
Ok may have solved this with iptables and DNATing, alerts from the remote sensor are now showing in the SOC alerts
